docs/driver-only-builds.md

```diff
@@ -240,3 +241,26 @@ removing builtin support (i.e. `MBEDTLS_DHM_C`).
 ### Limitations
 Support for deterministic derivation of a DH keypair
 (i.e. `PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_DERIVE`) is not supported.
+
+AEADs
+-----
+
+It is possible to have all AEADs operations provided only by a driver.
+
+More precisely you can:
+- enable desired PSA algorithm(s) and key type(s):
+  - `PSA_WANT_ALG_[CCM|GCM]` with `PSA_WANT_KEY_TYPE_[AES|ARIA|CAMELLIA]`
+  - `PSA_WANT_ALG_CHACHA20_POLY1305` with `PSA_WANT_KEY_TYPE_CHACHA20`;
+- enable `MBEDTLS_PSA_ACCEL_xxx` symbol(s) which correspond to the
+  `PSA_WANT_xxx` of the previous step;
+- disable builtin support of `MBEDTLS_[CCM|GCM|CHACHAPOLY]_C` algorithms and
+  key types `MBEDTLS_[AES|ARIA|CAMELLIA|CHACHA20]_C` for AEADs which are
+  accelerated.
+
+In such a build all AEADs operations requested through the PSA Crypto API
+(including those in TLS and X.509) will be performed by the provided driver.
+Of course direct calls to the disabled builtin modules
+(ex: `mbedtls_ccm_init()`, etc) won't be possible.
+
+If no other non-authenticated cipher is required, it is also possible to
```

Comment:
Wording: "If no unauthenticated cipher" instead. (Unauthenticated ciphers are not AEADs, while "other" suggests they are.)